.insomnia/Request/req_62d5f98e2f984cf4b2152bfdd863075d.yml

```diff
@@ -0,0 +1,47 @@
+_id: req_62d5f98e2f984cf4b2152bfdd863075d
+type: Request
+parentId: fld_eeb62a531f8441478a6d2f536021400d
+modified: 1676741836845
+created: 1676679154425
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/duplicates/{{
+  _.article1_id }}"
+name: Duplicates
+description: >+
+  Use this endpoint to resolve duplicates. Indicate the article to resolve by
+  specifying the article id in the path.
+
+
+  The following parameters are passed in the body:
+
+
+  - `duplicate_action`: An integer value that denotes the decision to apply to the duplicate. Can take the following values:
+  	- 1 = `USER_ACTION_YES` (article is a duplicate)
```

Comment:
Do the variable names (e.g. `USER_ACTION_YES`) have any significance to the API client user? If not, we can omit them to avoid confusion.